integration_test.py

```diff
@@ -93,3 +93,27 @@ def test_jobs(dcos_api_session):
         assert False
     except HTTPError as http_e:
         assert http_e.response.status_code == 404
+
+
+def test_packages(dcos_api_session):
+    pack_api = dcos_api_session.package
+    install_resp = pack_api.install('hello-world',
+                                    version='2.1.0-0.31.2')
+    installed_id = install_resp['appId']
+    dcos_api_session.marathon.wait_for_app_deployment(
+            installed_id, 1, True, False, 300)
+    packs = pack_api.list()
+    found = [p for p in packs['packages']
+             if p['appId'] == installed_id]
+    assert found
+    pack_api.uninstall('hello-world', app_id=installed_id)
+
+    pack_api.describe('jenkins')
```

Comment:
Is there no assert because these methods raise exceptions on failure? Looking through the implementation above it wasn't 100% obvious that `_make_request` couldn't just return empty when you expected output.